src/Directory.Build.props

```diff
@@ -1,7 +1,9 @@
 <Project>
+    <Import Project="Versions.props" />
```

Comment:
for our new projects, please remove uses of Import and shared props like this.
instead, add these to a Directory.Build.props or .targets, which is the modern way.